src/actions/network/index.test.js

```diff
@@ -0,0 +1,37 @@
+import { networkSet, networkStatusUpdated } from './index';
+import networks from '../../constants/networks';
+import { tokenMap } from '../../constants/tokens';
+import actionTypes from '../../constants/actions';
+
+describe('actions: network', () => {
+  let dispatch;
+
+  beforeEach(() => {
+    jest.resetModules();
+    dispatch = jest.fn();
+  });
+
+  describe('networkSet', () => {
+    it('should create networkSet action with name and token', () => {
+      const { name } = networks.testnet;
+      networkSet({ name })(dispatch);
+      expect(dispatch).toHaveBeenNthCalledWith(1, {
+        data: {
+          name,
+          token: tokenMap.BTC.key,
+          network: {},
```

Comment:
why do you store an empty object as a network in this action? this action only stores a value of a constant in a redux reducer.